src/client/canvas/renderer.ts

```diff
@@ -0,0 +1,242 @@
+import { Transform } from '../math/transform'
+import { Vector2 } from '../math/vector2'
+import { Appearance } from './appearance/appearance'
+import { BasicAppearance } from './appearance/basic_appearance'
+import { LineAppearance } from './appearance/line_appearance'
+import { ArrowGeometry } from './geometry/arrow_geometry'
+import { CircleGeometry } from './geometry/circle_geometry'
+import { LineGeometry } from './geometry/line_geometry'
+import { MarkerGeometry } from './geometry/marker_geometry'
+import { PolygonGeometry } from './geometry/polygon_geometry'
+import { TextGeometry } from './geometry/text_geometry'
+import { Shape } from './object/shape'
+
+// TODO Monica fix typing here
+export type Scene = Array<any>
```

Comment:
Will be addressed in the next PR :) Scene will be converted to a proper Object2d